src/com/egs/suzy/task1/FileHandler.java

```diff
@@ -0,0 +1,28 @@
+import java.io.*;
+
+public class FileHandler {
+    public static void CountHellos(File in, File out){
+        try(BufferedReader reader = new BufferedReader(new FileReader(in));
+        BufferedWriter writer = new BufferedWriter(new FileWriter(out, true))){
+            String[] arr;
+            String line = reader.readLine();;
+            int count = 0, lineNum = 0;
+            while(line!=null){
+              arr = line.split("\\s");
+              for(String s : arr){
+                  if(s.equalsIgnoreCase("hello"))  count++;
```

Comment:
even if there is a line of code after if, while, for etc, it still good to have braces { }